lib/internal/Magento/Framework/Api/AbstractExtensibleObject.php

```diff
@@ -12,6 +12,7 @@
  *
  * @SuppressWarnings(PHPMD.NumberOfChildren)
  * @api
+ * @codeCoverageIgnore
```

Comment:
```suggestion
 * phpcs:disable Magento2.Classes.AbstractApi
```

Reply:
**done**

@lenaorobei , could you please review? Thank you!